server/backend/config.py

```diff
@@ -0,0 +1,84 @@
+import os
+
+def is_truthy(v):
```

Comment:
Curious to why this function is here?

Reply:
It's used shortly after to coerce "truthy" strings specified as config settings (e.g., "yes", "y", "true", "t", etc.) to a boolean True, and anything else to False. I could put it in a separate module, but since it's only used in this config module I figured it'd make sense to leave it there. If you'd rather I factor it out to a utility library, that's fine with me.

Reply:
> I could put it in a separate module, but since it's only used in this config module I figured it'd make sense to leave it there. If you'd rather I factor it out to a utility library, that's fine with me.

Don't think it's needed. Feel free to leave it in the file.